game/scripts/player/player.gd

```diff
@@ -54,3 +54,13 @@ func _on_weapon_force_timer_timeout():
 func inventory_input() -> void:
 	if Input.is_action_just_pressed("inventory"):
 		toggle_inventory.emit()
+
+## Adds world_item to inventory
+func pick_up_world_item(world_item: WorldItem) -> bool:
```

Comment:
i think moving this sort of stuff to an inventory node which is a child of the player makes sense once we have a better handle on next steps.

Reply:
Yea then the world_item can just add itself to the Entity's (Player into his case) inventory

Reply:
updated